library/src/main/java/com/chuckerteam/chucker/internal/support/OkHttpUtils.kt

```diff
@@ -60,3 +63,17 @@ private val Headers.containsGzip: Boolean
     get() {
         return this["Content-Encoding"].equals("gzip", ignoreCase = true)
     }
+
+private val supportedEncodings = listOf("identity", "gzip")
+
+internal val Headers.hasSupportedContentEncoding: Boolean
+    get() = get("Content-Encoding")
+        ?.takeIf { it.isNotEmpty() }
+        ?.let { it.toLowerCase(Locale.ROOT) in supportedEncodings }
```

Comment:
nit: In another part of the codebase we use `Locale.US`. I believe we should switch to `.ROOT` also the other but I'm unsure.

Reply:
The other code uses it for formatting. Leaving `US` there is fine.

https://github.com/ChuckerTeam/chucker/blob/b3761b7ceca25386624520fe49a5cbdb713b4a8f/library/src/main/java/com/chuckerteam/chucker/internal/support/FormatUtils.kt#L52